_includes/specials/taproot/en/06-multisignatures.md

```diff
@@ -0,0 +1,147 @@
+required, but multisignatures are a constant small size.  The smallest
+effective multisig policy (1-of-2) requires more space than a
+multisignature policy that can involve thousands of signers.  The
+decrease in size leads to direct reduction in fees for the
+multisignature users and an indirect reduction in fees for all users as
+the same amount of demand for confirmed transactions can be fulfilled
+using a smaller amount of block space.
+
+{:.center}
+![Plot showing the savings for multisignatures compared to multisig](/img/posts/2021-07-multisignature-savings.png)
+
+The second major benefit is improved privacy.  Each use of multisigs is
+distinctively recorded to the block chain where surveillance companies
+can use them to make informed guesses about the wallet history and
+current balance of individual users.  For example, looking at block
+692,039, we can distinguish not just the multisig spends from the
+single-sig spends but also distinguish between different set sizes and
+thresholds for the multisigs.
+
+{:.center}
+![Illustration of the lack of witness fungibility in current blocks](/img/posts/2021-07-multisig-unfungible.png)
+
+By comparison, a third party looking only at block chain data can't tell
+that a spender used a multisignature.  When a multisignature is used for
+a keypath spend, it is indistinguishable from single-sig spends.  If all
+single-sig and multisigs in the block above were switched to P2TR
+keypath spends, only a few exotic spends would be distinguishable by
+their scripts (and even some of those might be able to sometimes use
+keypath spends).
+
+{:.center}
+![Illustration of how fungibile witnesses could be ideally](/img/posts/2021-07-multisignature-fungible.png)
+
+### Using multisignatures
+
+We're aware of three schnorr-based multisignature schemes designed
+specifically for Bitcoin, all members of the [MuSig][topic musig]
+family:
+
+- **MuSig** (also called MuSig1), which should be simple to implement
+  but which requires three rounds of communication during the signing
+  process.
+
+- **MuSig2**, also simple to implement and which eliminates one round of
+  communication and allows another round to be combined with key
+  exchange so that it's possible to use a somewhat similar signing
+  process to what we use today with script-based multisig.  This does
+  require storing extra data and ensuring your signing software or
+  hardware can't be tricked into unknowingly repeating part of the
+  signing session.
+
+- **MuSig-DN** (Deterministic Nonce), significantly more complex to
+  implement and its communication between participants can't be combined
+  with key exchange.  However, it's not vulnerable to the repeated
+  session attack.
+
+All signers have to agree on the protocol to use, so there will likely
+be a network effect where almost all implementations use the same
+protocol.  The authors of the MuSig proposals [suggest][nick ruffing
```

Comment:
Maybe, although I think we're a very long way off being able to create generalised multisignatures across an arbitrary set of wallet implementations. I would have thought that most people using multisignature initially would just use a single wallet for all of the signers, which would mean that they could choose any multisignature scheme they want.

Reply:
> I think we're a very long way off being able to create generalised multisignatures across an arbitrary set of wallet implementations.

Interesting!  I thought the opposite.  All the work projects like HWI have put into creating compatibility shims, plus the need for only small upgrades to PSBT, seems to me to imply that all that's left to do is for actual signing hardware/software to implement MuSigWhatever.  Surely that's two weeks work at most 😀

> I would have thought that most people using multisignature initially would just use a single wallet for all of the signers, which would mean that they could choose any multisignature scheme they want.

That probably is true.  But network effects travel back through time through the anticipation of those effects.  I think there are few wallet developers who want to spend their time implementing a different protocol than the protocol they expect to become widely used in the long term.

Anyway, I'll see if I can tone down the statement in the text.

Reply:
New wording looks good to me. Thanks!